infoportal-client/src/features/Meta/useMetaData.ts

```diff
@@ -62,7 +62,8 @@ export const useMetaDashboardData = (data: Seq<IKoboMeta>) => {
       status: {
         icon: 'check_circle',
         label: m.status,
-        getValue: _ => _.status,
+        getValue: _ => _.status ?? DataFilter.blank,
+        addBlankOption: true,
```

Comment:
Didn't we already merge that? 😄
https://github.com/alexandreannic/infoportal/pull/30/files